Recent change in message_lite.h breaks build of generated code

**What version of protobuf and what language are you using?**
Version: 8dadfda1e3c270458dbff0a2aff3218632890882
Language: C++

**What operating system (Linux, Windows, ...) and version?**
Linux

**What runtime / compiler are you using (e.g., python version or gcc version)**
GNU 7.3.0

**What did you do?**
Steps to reproduce the behavior:
1. Build protobuf with default options in above commit
2. Generate message code with `protoc`
3. Compile message code

**What did you expect to see**
Successful compilation

**What did you see instead?**
```shell
In file included from /usr/local/include/google/protobuf/wire_format_lite_inl.h:45:0,
                 from /usr/local/include/google/protobuf/map_type_handler.h:35,
                 from /usr/local/include/google/protobuf/map.h:49,
                 from /usr/local/include/google/protobuf/generated_message_table_driven.h:34,
                 from /home/travis/build/proio-org/cpp-proio/build/proio/model/eic/eic.pb.h:26,
                 from /home/travis/build/proio-org/cpp-proio/build/proio/model/eic/eic.pb.cc:4:
/usr/local/include/google/protobuf/message_lite.h:49:10: fatal error: google/protobuf/stubs/strutil.h: No such file or directory
```

I see that strutil.h is indeed not installed.  In googling around I see people saying that it is not supposed to be installed.  However, a very recent commit (0894e07536af88065d462cdc9d8e807c0723ef4d) has added a dependency to it in installed header `message_lite.h`:
https://github.com/protocolbuffers/protobuf/blob/8dadfda1e3c270458dbff0a2aff3218632890882/src/google/protobuf/message_lite.h#L49

**Anything else we should know about your project / environment**
It is a standard, clean environment in a container, with protobuf built from the master branch.

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -77,6 +77,7 @@ nobase_include_HEADERS =                                         \
   google/protobuf/stubs/status.h                                 \
   google/protobuf/stubs/stl_util.h                               \
   google/protobuf/stubs/stringpiece.h                            \
+  google/protobuf/stubs/strutil.h                                \
   google/protobuf/stubs/template_util.h                          \
   google/protobuf/any.pb.h                                       \
   google/protobuf/api.pb.h                                       \
@@ -195,7 +196,6 @@ libprotobuf_lite_la_SOURCES =                                  \
   google/protobuf/stubs/stringprintf.h                         \
   google/protobuf/stubs/structurally_valid.cc                  \
   google/protobuf/stubs/strutil.cc                             \
-  google/protobuf/stubs/strutil.h                              \
   google/protobuf/stubs/time.cc                                \
   google/protobuf/stubs/time.h                                 \
   google/protobuf/arena.cc                                     \

